tgui/packages/tgui/components/Stack.tsx

```diff
@@ -18,16 +18,18 @@ import {
 type StackProps = FlexProps & {
   vertical?: boolean;
   fill?: boolean;
+  compressed?: boolean;
 };
 
 export const Stack = (props: StackProps) => {
-  const { className, vertical, fill, ...rest } = props;
+  const { className, vertical, fill, compressed, ...rest } = props;
```

Comment:
Stack's feature is automatic margin, so this seems like an extremely controversial solution. m={0} is much simpler

Reply:
I don't think you should use Flex instead of Stack. Since the latter is essentially an improved Flex